src/main.js

```diff
@@ -1,3 +1,21 @@
 const core = require('@actions/core');
+const { action_library } = require('./action_library');
 
-core.info('Hello!');
+let library = new action_library();
+
+let scope = core.getInput('scope');
+let createOutputVariables = core.getInput('create-output-variables') == 'true';
+
+let inputFilePath = core.getInput('input-file');
+let inputYaml = library.getFileYaml(inputFilePath);
+
+let environmentYaml = library.getCurrentEnvironmentVars();
+let environmentDictionary = library.buildEnvironmentDictionary(scope, inputYaml, environmentYaml);
+
+console.log('Scoped Variables:', environmentDictionary);
+for (envVar in environmentDictionary) {
+  library.setEnvironmentVar(`${envVar}`, environmentDictionary[envVar]);
+  if (createOutputVariables) {
+    action_library.setOutputVar(`${envVar}`, environmentDictionary[envVar]);
```

Comment:
Should this be using `library` like the others are?  And same question about using the template string...why?
```suggestion
    library.setOutputVar(`${envVar}`, environmentDictionary[envVar]);
```

Reply:
👍🏽. This was part of what I added this morning...should have looked more closely to what I was using to make the call.